src/o365/graph/commands/teams/teams-add.ts

```diff
@@ -0,0 +1,197 @@
+import auth from '../../GraphAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import Utils from '../../../../Utils';
+import GraphCommand from '../../GraphCommand';
+import * as request from 'request-promise-native';
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  groupId?: string;
+  name?: string;
+  description?: string;
+}
+
+class GraphTeamsAddCommand extends GraphCommand {
+  public get name(): string {
+    return `${commands.TEAMS_ADD}`;
+  }
+
+  public get description(): string {
+    return 'Add a new Microsoft Teams team.';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    return telemetryProps;
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    auth
+      .ensureAccessToken(auth.service.resource, cmd, this.debug)
+      .then((): request.RequestPromise => {
+        return args.options.groupId ? this.CreateGroupTeamRequest(cmd,args) : 
+                                      this.CreateTeamRequest(cmd,args);
+      })
+      .then((res: any): void => {
+        // get the teams id from the response header.
+        var teamsId = this.GetTeamsIdFromResponse(res, args);
+        cmd.log({Team:teamsId});
```

Comment:
Is there a reason why we're wrapping the retrieved value in a custom object?

Reply:
Hi @waldekmastykarz, We are getting the TeamsId from the response header (res.headers.location). Thought it would be easier to understand if we have print out the details.

Reply:
I'm all for printing out the details, but I'd prefer it to be a plain string rather than a JSON object.

Reply:
We are getting the below response in the header: res.headers.location - /teams('f9526e6a-1d0d-4421-8882-88a70975a00c')/operations('6cf64f96-08c3-4173-9919-eaf7684aae9a').  Currently we are retrieving only the Team Id and displaying it in a JSON format. Can I amend it to display the full or just a string format of the Team Id (guid)?

Reply:
I'd say, let's display just the ID as plain text, eg `cmd.log(teamId)`

Reply:
Thanks @waldekmastykarz, I have amended to display the Team Id as a string.